javascript/src/jobProcessor.ts

```diff
@@ -82,15 +102,18 @@ export const jobProcessor = async (opts?: NatsOpts) => {
    * To gracefully shutdown see stop method.
    */
   const start = async (def: JobDef) => {
+    let extendAckTimer: NodeJS.Timer | undefined
     debug('JOB DEF %O', def)
     const pullInterval = def.pullInterval ?? ms('1s')
     const backoff = def.backoff ?? ms('1s')
     const batch = def.batch ?? 10
+    const autoExtendAck = def.autoExtendAck ?? true
```

Comment:
I wonder if this might be a clearer name: `autoExtendAckTimeout`. Thoughts?

Reply:
> I wonder if this might be a clearer name: `autoExtendAckTimeout`. Thoughts?

Of course, I was just trying to avoid this option being the `black sheep` one with a very long name, but the longer name does explain better the purpose, I will update

Reply:
@dubiousdavid updated